### ISSUE TITLE
--recurse=yes doesn't work

### ISSUE THREAD

**Issue by tssm** [vim-php/phpctags#25]
I need to generate a tags file for each of my projects using phpctags from the command line.

```
phpctags .
```

works.

```
phpctags -f php.tags .
```

works, but

```
phpctags -R .
```

or

```
phpctags --recurse=yes .
```

does't work, i. e., no tags file is generated.

```
phpctags --recurse=no .
```

also works. I'm using the version cloned from GitHub.

**Comment by INTPnerd** [comment]
I am getting the same result.

**Comment by INTPnerd** [comment]
I got it to work. I used the github web interface to view and then download the raw file PHPCtags.class.php from the following pull request: https://github.com/vim-php/phpctags/pull/42

Of course after downloading the file I had to rebuild using this source. Then the following command will generate a tags file for me:
`phpctags -R --memory=-1 .`
Note that this still does not work for me if I exclude the `--memory=-1` part. I did confirm that this command does not work for me until I rebuild with the pull request.

I am not sure, but looking at the pull request it appears to fix an issue with global variables in your code, so removing global variables or references to them from a non-global place, may also be another workaround. The pull request has been recently added to the 0.5.2 milestone, so hopefully this will get a release soon.
